### ISSUE TITLE
Missing alt on image on tags browse admin page

### ISSUE THREAD

**Issue by aviannamiller** [omeka/Omeka#1057]
There is an image included in instructional text under Editing Tags on the Browse Tags page. It is inserted into the CSS as a background image and is thus missing alt text:
<img width="780" alt="Diagram of tag parts nestled under Editing Tags subheading on Browse Tags page. Browser dev tools are open next to the image, showing its source as a background image inserted into CSS" src="https://github.com/user-attachments/assets/0c6c70ff-606d-4819-ac73-60956b88a0a2" />

**Comment by kimisgold** [comment]
A fix for this is ready for testing on the `a11y/tag-example-alt` branch. It converts the example from a background image to an actual `img` element and drafts some alt text: 

> Example of editable tag, labeled "1" for the associated record count, "2" for editable name, and "3" for delete action.

**Comment by aviannamiller** [comment]
Sounds great, thanks. However, I think the use of quotes in the text has truncated the alt after "labeled", putting the rest of it in as attributes.

**Comment by kimisgold** [comment]
Got it, quotes should be fixed now.

**Comment by aviannamiller** [comment]
Perfect! All set here.
